specs/sql/README.md

```diff
@@ -480,7 +451,60 @@ subsequent inserts, resulting in gaps in the `ROWID` sequence. Tableland
 guarantees that automatically chosen `ROWID`s will be increasing but not
 that they will be sequential.
 
-#### DELETE
+### ALTER TABLE
+
+The `ALTER TABLE` command allows the following alterations of an
+existing table: renaming a column, adding a column, and dropping a
+column.
+
+#### Structure
+
+``` sql
+ALTER TABLE table_name *action*
+```
+
+where action is one of:
+
+``` sql
+-- For renaming a column
+RENAME [ COLUMN ] *column_name* TO *new_column_name*
+
+-- For adding a column
+ADD [ COLUMN ] *column_name* *data_type* [ *column_constraint* [,  ... ] ]
+
+-- For dropping a dolumn
+DROP [ COLUMN ] *column_name* 
+```
+
+#### Details
+
+The `ADD COLUMN` syntax is used to add a new column to an existing
+table. The new column is always appended to the end of the list of
+existing columns. The new column may take any of the forms permissible
+in a [`CREATE TABLE`](#create-table) statement, with the following
+restrictions:
```

Comment:
Good point! Yeah, not sure if the correct place is the SQL spec because that's more protocol related. @dtbuchholz what's a good place to say who is allowed to execute ALTER TABLE statements?

Reply:
@brunocalza perhaps on this page at the bottom: https://docs.tableland.xyz/playbooks/sql/create? 

Alternatively, we could create a dedicated page for `ALTER TABLE`. But, it's probs just easier to throw it in at the bottom of the page linked above.

Reply:
@dtbuchholz @joewagner  i added the [section](https://docs-edai3zpb9-tableland.vercel.app/playbooks/sql/create#altering-an-existing-table) lmk how it looks

also, question: do I need to copy and paste the contents of `specs/sql/README.md` to `docs/specs/sql/README.md` to have the ALTER TABLE available to the docs?

Reply:
Looks good to me.  The only nit is that since "value" is a reserved word in some sql specs (not ours), it looks like our markdown interpreter is not giving it the correct color in the example.
Specifically, `ALTER TABLE my_table RENAME COLUMN val to value;` is showing `value` as blue, when it should be gray.  Would it make sense to use a different column name identifier?